app/Project.swift

```diff
@@ -0,0 +1,72 @@
+import ProjectDescription
+
+func tuistAppDependencies() -> [TargetDependency] {
+    [
+        .external(name: "Path"),
+        .external(name: "TuistSupport"),
+        .external(name: "TuistCore"),
+        .external(name: "TuistServer"),
+        .external(name: "TuistAutomation"),
+        .external(name: "Mockable"),
+        .external(name: "XcodeGraph"),
+    ]
+}
+
+let project = Project(
+    name: "Tuist",
+    settings: .settings(
+        debug: [
+            "SWIFT_ACTIVE_COMPILATION_CONDITIONS": "$(inherited) MOCKING",
```

Comment:
What do you think if we enable [strict concurrency](https://developer.apple.com/documentation/swift/adoptingswift6) now to avoid migration pain down the road?

Reply:
After turning it on, there are some build issues related to passing the flag (not the concurrency checking). Let's do this later.